pkgs/by-name/js/jsql-injection/package.nix

```
@@ -0,0 +1,90 @@
+{
+  lib,
+  jre,
+  unzip,
+  stdenv,
+  fetchurl,
+
+  makeWrapper,
+  makeDesktopItem,
+  makeBinaryWrapper,
+
+  copyDesktopItems,
+  desktopToDarwinBundle,
+}:
+
+let
+  icon = fetchurl {
+    url = "https://www.kali.org/tools/jsql/images/jsql-logo.svg";
+    hash = "sha256-qjr+9gMGSdYqDT2SZFt4i3vISrMjqvAlBTt7U6raTLc=";
+  };
+
+  pname = "jsql-injection";
+  version = "0.111"; # ! Latest = v0.111
+in
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivationNoCC {
```
don't need a C compiler here

---

My bad it's `stdenvNoCC.mkDerivation`

---

Tested it and it works, Thanks so much for your help and effort getting me to understand how to do this properly